src/index.js

```diff
@@ -1 +1,37 @@
+/* eslint-disable no-console */
 'use strict';
+'use strict';
```

Comment:
The 'use strict' directive is duplicated. You only need one 'use strict' directive at the top of the file. Consider removing the second occurrence.